src/elona/ctrl_file.cpp

```diff
@@ -1561,6 +1631,10 @@ void ctrl_file(FileOperation file_operation)
 
 void ctrl_file(FileOperation2 file_operation, const fs::path& filepath)
 {
+    ELONA_LOG("save.ctrl_file")
+        << "ctrl_file2 " << static_cast<int>(file_operation) << " mid: " << mid
+        << " filepath: " << filepath.string();
```

Comment:
At least in this project, there is no reason to use `boost::filesystem::string()` as this method encodes internal content *in OS dependent encoding*. It means, UTF16 filepath in Windows will be converted to ANSI (in Japanese, Shift-JIS). Instead, always use `filepathutil::to_utf8_path()` to convert filepath to `std::string` *in UTF8*.